app/graphql/albums/resolvers.js

```diff
@@ -1,4 +1,14 @@
-const { findAlbumById, findPhotosByAlbumId } = require('../../services/albums');
+const { findAlbumById, findPhotosByAlbumId, findAlbums } = require('../../services/albums'),
+  { orderArrayByField } = require('../../utils/arrays');
+
+const DEFAULT_LIMIT = 3,
```

Comment:
make a constants file to store this values and values similar to this

Reply:
In the next card I put these values inside the albums query, i.e, `albums(offset: Int = 0, limit: Int = 3, orderBy: AlbumFields = id, filter: String): [Album!]!`